yeonjae22/ElevenStreetSample/app/src/main/java/com/exam/elevenstreet/view/product/ProductFragment.kt

```diff
@@ -29,7 +29,7 @@ class ProductFragment : Fragment() {
         binding.productDeliveryTv.text = arguments!!.getString("productDelivery")
 
         binding.btnBack.setOnClickListener {
-            (activity as ProductActivity).back(this)
+            (activity as ProductActivity).onBackPressed()
```

Comment:
```suggestion
            requireActivity().onBackPressed()
```
activity의 코드를 변경한 의도에 맞게 변경해주세요.

Reply:
수정했습니다